src/Component/UserBox.tsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import styled from 'styled-components';
+//import { useMessageState, useMessageDispatch } from '../Context/MessageContext';
+
+
+const UserBox = ( ) => {
+    //const state = useMessageState();
+    //const dispatch = useMessageDispatch();
+
+    const onClick = ( e : React.FormEvent ): void => {
+        e.preventDefault(); //여기에 current 유저 토글기능 넣기
+    }
+
+    return<>
+        <UserField onClick={onClick}>
+            <UserProfile>
+                <ProfileImg/>
+                <ProfileText>
+                    <UserName>{"Nick Wilde"}</UserName>
+                    <UserComment>{"Way to hustle, bud!"}</UserComment>
+                </ProfileText>
+            </UserProfile>
+        </UserField>
+    </>    
```

Comment:
여기에선 fragment를 굳이 사용할 이유가 없어 보이네요! 그리고 네이밍 아주 좋습니다!

```suggestion
    return
        <UserField onClick={onClick}>
            <UserProfile>
                <ProfileImg/>
                <ProfileText>
                    <UserName>{"Nick Wilde"}</UserName>
                    <UserComment>{"Way to hustle, bud!"}</UserComment>
                </ProfileText>
            </UserProfile>
        </UserField>
```